charts/pdf-generator/templates/deployment.yaml

```
@@ -0,0 +1,59 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "name" . }}
+  namespace: {{ .Values.namespace }}
```

Sorry my bad! I just remembered that it's possible to use a built in variable in helm to control namespace, we can then use: `helm install --namespace whatever` to control the namespace it should be installed

```suggestion
  namespace: {{ Release.Namespace }}
```